doc/version_history.rst

```diff
@@ -6,6 +6,16 @@
 Version History
 ###############
 
+v1.1.3
+------
+
+* Add the new commands to set the emergency acceleration and jerk in the run time.
+
+Requires:
+
+* ts_rotator_controller 1.6.5
+* ts_xml 21.0.0
```

Comment:
Please make sure that the next ts_xml version indeed is 21.0.0. According to https://confluence.lsstcorp.org/display/LSSTCOM/Cycle+35+Upgrade it will be 20.2.0.

Reply:
Updated in the commit: [3ed04af](https://github.com/lsst-ts/ts_mtrotator/pull/86/commits/3ed04af203aa5ccebcfea3464dfa1b4652aea1f6). Thanks!